integration-cli/docker_cli_build_test.go

```diff
@@ -669,6 +669,64 @@ func TestBuildCacheADD(t *testing.T) {
 	logDone("build - build two images with remote ADD")
 }
 
+func TestBuildLastModified(t *testing.T) {
+	name := "testbuildlastmodified"
+	defer deleteImages(name)
+
+	server, err := fakeStorage(map[string]string{
+		"file": "hello",
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.Close()
+
+	var out, out2 string
+
+	if _, out, err = buildImageWithOut(name,
+		fmt.Sprintf("FROM busybox\nADD %s/file /\nRUN ls -le /file", server.URL),
+		false); err != nil {
+		t.Fatal(err)
+	}
+
+	originMTime := regexp.MustCompile(`root.*/file.*\n`).FindString(out)
```

Comment:
How about we `stat /file` instead of `ls -el`?

I can't parse what originMTime/newMTime looks like here. Is the entire variable like this: `root     root             7 Thu May 22 23:22:27 2014 /file`? If so, that's not really a mtime string and we should probably `RUN stat /file` –though busybox doesn't have `/bin/stat`, we should also figure that out.